crates/libs/metadata/src/reader/mod.rs

```diff
@@ -218,14 +216,14 @@ impl<'a> Reader<'a> {
         }
         Self { files, types, nested }
     }
-    pub fn tree(&self, root: &str) -> Option<Tree> {
+    pub fn tree(&'a self, root: &'a str, exclude: &[&str]) -> Option<Tree> {
```

Comment:
These lifetime annotations seems wrong, but I guess if it works it's fine.

Reply:
Yeah, that's the lifetime annotations the Rust compiler's error message suggested. 🤷